### ISSUE TITLE
Broken shader on satisfactory.

### ISSUE THREAD

**Issue by UltraWelfare** [doitsujin/dxvk#1838]
Satisfactory has a broken highlight shader (It should be a white outline), it renders incorrectly like boxes.
Images on the broken render is [here](https://imgur.com/a/XqIxfLu).

I must mention that I'm using the Epic Games Store version, not the steam one and I'm also using Lutris to launch the game.

I was not able to get an apitrace even though I followed all the steps on the Lutris documentation on apitracing.
Copying the DLL's both on the launcher executable folder and on the game folder doesn't seem to generate any trace (I also used `find` to check if it was hidden anywhere). 
I tried tracing both with DXVK enabled and disabled.
My last try was manually importing the game executable (instead of opening it through EGS) on lutris and doing everything again but it didn't seem to generate a trace. I suspect this has to do with EGS but I do not have the technical understanding to do this on my own so I'm sorry for that.

The shader seems to be rendered correctly by disabling DXVK ( I guess it launches with wined3d).

I tried multiple wine versions (TKG, wine-staging, etc) and all of them have the same problem however in system information I will only mention the TKG one as it is the primary wine I use.
 
### Software information
Name of the game: Satisfactory EGS Version
Launcher: Lutris

### System information
- GPU: Radeon 530 (Laptop with dedicated GPU)
- Driver: amdgpu
- Wine version: 5.22.r5.g991f2e8e ( TkG Staging Esync Fsync )
- DXVK version: 1.7.2L

### Apitrace file(s)
- Was not able to provide.

### Log files
- d3d9.log: empty
- d3d11.log: https://pastebin.com/thaVbVcs
- dxgi.log: https://pastebin.com/4iUkdnnu

**Comment by K0bin** [comment]
Does amdgpu mean AMDGPU Pro or are you referring to the kernel driver?

Can you reproduce the issue with one of the other AMD drivers?

**Comment by UltraWelfare** [comment]
> Does amdgpu mean AMDGPU Pro or are you referring to the kernel driver?
> 
> Can you reproduce the issue with one of the other AMD drivers?

I assume I refer to the kernel driver...

I used `lspci -v` to get that information 
```
01:00.0 Display controller: Advanced Micro Devices, Inc. [AMD/ATI] Topaz XT [Radeon R7 M260/M265 / M340/M360 / M440/M445 / 530/535 / 620/625 Mobile] (rev c1)
        Subsystem: Dell Radeon 530
        Flags: bus master, fast devsel, latency 0, IRQ 132
        Memory at c0000000 (64-bit, prefetchable) [size=256M]
        Memory at d0000000 (64-bit, prefetchable) [size=2M]
        I/O ports at e000 [size=256]
        Memory at d0200000 (32-bit, non-prefetchable) [size=256K]
        Expansion ROM at d0240000 [disabled] [size=128K]
        Capabilities: <access denied>
        Kernel driver in use: amdgpu
        Kernel modules: amdgpu```

As far as I know AMDGPU Pro is not supported for my GPU.

**Comment by niobium93** [comment]
His d3d11.log says he's using radv with aco from mesa 20.2.3.

**Comment by UltraWelfare** [comment]
I didn't even know about `radv` `amdvlk` etc...
I just tried `amdvlk` and the issue seems to be gone, with at least 20 frames lower tho :(..

I also tried radv with ACO disabled and the issue seems to persist so I assume it has to do with radv.
Sorry for opening the bug thread here, I was not aware of some things.
A last question would be: where is the correct place to report this bug ?

Thanks again for your time!
